### ISSUE TITLE
Improve accessibility support

### ISSUE THREAD

**Issue by chanikag** [wso2/ballerina-plugin-vscode#620]
### Description

Encountered few areas after evaluating BI vscode extension against VPAT.

1.1.1 Non-text Content - Partially supported - Incorporates alternative text for the majority of its UI elements. But it requires additional enhancements like ARIA labels or descriptive text to ensure full accessibility.
1.3.1 Info and Relationships - Partially supported - Most of the elements like panels and trees support programmatic structure. However, it requires additional enhancements like ARIA labels to fully support.
2.1.1 Keyboard - Partially supported - Most of the UI components support keyboard navigation. But some of the fields are missing.
2.1.2 No Keyboard Trap - Partially supported - Most of the UI components support users to move focus freely between UI elements without getting stuck. But some modals can trap focus unless explicitly closed.
2.5.3 Label in Name - Partially supported - Most controls with visible labels reflect those in the accessible name, but some are lacking aria-label.
4.1.2 Name, Role, Value - Partially supported - Most UI components expose appropriate name and role properties, but lack explicit ARIA roles and programmatically exposed names.

This is encountered while doing https://github.com/wso2/product-ballerina-integrator/issues/417 VPAT report for BI
